motionplan/tpspace/ptgIK.go

```diff
@@ -173,3 +175,11 @@ func (ptg *ptgIK) Trajectory(alpha, dist float64) ([]*TrajNode, error) {
 func (ptg *ptgIK) Transform(inputs []referenceframe.Input) (spatialmath.Pose, error) {
 	return ptg.ptgFrame.Transform(inputs)
 }
+
+func (ptg *ptgIK) arcDist(inputs []referenceframe.Input) float64 {
+	dist := 0.
+	for i := 1; i < len(inputs); i += 2 {
+		dist += (inputs[i].Value - defaultMinPTGlen)
```

Comment:
In this case aren't inputs in the encoded format? What does it mean to do this calc?

Reply:
A `ptgIK` is in the bucket of "thing that understands what its inputs mean", rather than "inputs are opaque floats". So it's allowed to do this.

This is just a convenience function to easily sum the distance values on a series of PTGs.

Reply:
Maybe I'm not understanding the structure of the input then.  Because this still doesn't track for me.  I thought they were length 3 for PTGs

Reply:
PTG inputs are length 3 to a PTG input-enabled base, where the first of the three inputs is "which PTG family do we use?"

This code is after the family has been selected, leaving only the two remaining inputs; alpha and distance.